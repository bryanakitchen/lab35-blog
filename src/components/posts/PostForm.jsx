import React from 'react';
import PropTypes from 'prop-types';

export default function PostForm() {

  return (
    // <form onSubmit={handleSubmit}>
    <form>
      <input 
        type="text" 
        placeholder="Title" 
      />
      <br />
      <textarea 
        placeholder="Body" 
        rows="10" 
        cols="50"></textarea>
      <br />

      <button>Post</button>
    </form>
  );
}

PostForm.propTypes = {

};
