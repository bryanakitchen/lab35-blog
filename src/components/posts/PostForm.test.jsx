import React from 'react';
import { render, cleanup } from '@testing-library/react';
import PostForm from './PostForm';

describe('PostForm component', () => {
  afterEach(() => cleanup());
  it('renders PostForm', () => {
    const { asFragment } = render(<PostForm />);
    expect(asFragment()).toMatchSnapshot();
  });
});
