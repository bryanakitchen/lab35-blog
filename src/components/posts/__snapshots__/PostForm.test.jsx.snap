// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostForm component renders PostForm 1`] = `
<DocumentFragment>
  <form>
    <input
      placeholder="Title"
      type="text"
    />
    <br />
    <textarea
      cols="50"
      placeholder="Body"
      rows="10"
    />
    <br />
    <button>
      Post
    </button>
  </form>
</DocumentFragment>
`;
